site/_data/master-toc.yml

```diff
@@ -1,30 +1,27 @@
 toc:
-  - title: Setting up
+  - title: Basics
     subfolderitems:
-      - page: Getting started
+      - page: Overview
         url: /index.html
-      - page: Config
-        url: /sonobuoy-config
       - page: Checking Results
         url: /results
-      - page: Snapshot layout
-        url: /snapshot
-  - title: Use cases
+  - title: Plugins
+    subfolderitems:
+      - page: Plugins
```

Comment:
It might be better to avoid the repetition here of `Plugins` being the top level item, and then the first item in the list. Perhaps the name could be `Overview` or something?